hibernate-envers/src/test/java/org/hibernate/envers/test/tools/TestTools.java

```diff
@@ -27,11 +27,9 @@
  */
 public class TestTools {
 	public static <T> Set<T> makeSet(T... objects) {
-		final Set<T> ret = new HashSet<T>();
+		final Set<T> ret = new HashSet<T>(objects.length);
```

Comment:
btw, the above usage is not totally correct for it doesn't take into consideration of 'load factor'. Suppose load factor is 0.75, you might specify as `new HashSet<>(objects.length / 0.75)`.
Guava library provides util method for this, i.e. `Sets.withExpectedSize(objects.length)`. see https://guava.dev/releases/23.0/api/docs/src-html/com/google/common/collect/Sets.html#line.252

Reply:
Yes, what would be better is the constructor with two parameters including the loadfactor itself.

Reply:
Recently I found Hibernate has provided a util method for this: `org.hibernate.internal.util.collections.CollectionHelper.mapOfSize(int size)` already